tests/copyright/policy

```diff
@@ -22,6 +22,6 @@ modules/native
  GPLv2+_SSL
 tests
 modules/java/(tools|[^/]*$)
-modules/java-modules/(dummy|elastic|hdfs|http|kafka|[^/]*$)
+modules/java-modules/(dummy|elastic|elastic-v2|hdfs|http|kafka|[^/]*$)
```

Comment:
Hmm... we should add every new module to this file?
That will always be a problem...